### ISSUE TITLE
Out of memory error

### ISSUE THREAD

**Issue by StarPool** [xumacoin/xuma-core#6]
OS: Ubuntu 16.04

Excute `xuma-cli getblocktemplate` show error

```
error: {"code":-7,"message":"Out of memory"}
```

`xuma-cli getinfo`

```
{
    "version" : 1010100,
    "protocolversion" : 10100,
    "walletversion" : 61000,
    "balance" : 0.50506318,
    "Darksend_balance" : 0.00000000,
    "blocks" : 82391,
    "timeoffset" : 0,
    "connections" : 12,
    "proxy" : "",
    "difficulty" : 37636.61008632,
    "testnet" : false,
    "keypoololdest" : 1522820148,
    "keypoolsize" : 1001,
    "paytxfee" : 0.00000000,
    "relayfee" : 0.00010000,
    "staking status" : "Staking Not Active",
    "errors" : ""
}
```

**Comment by makumori** [comment]
Please try version 1.1.2.

**Comment by makumori** [comment]
This was resolved with 1.1.2, closing the issue.
